core-services/prow/02_config/_config.yaml

```diff
@@ -1871,6 +1872,17 @@ tide:
     - oVirt/cluster-api-provider-ovirt
     - oVirt/csi-driver
     - oVirt/csi-driver-operator
+  - labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift/assisted-test-infra
```

Comment:
We prefer adding the repo to an existing query (-ies) over creating new ones, especialy for repos in `openshift`.

Reply:
Fixed.